src/seedu/addressbook/commands/Command.java

```diff
@@ -39,10 +39,8 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     /**
      * Executes the command and returns the result.
      */
-    public CommandResult execute(){
-        throw new UnsupportedOperationException("This method should be implement in child classes");
-    }
 
```

Comment:
Don't leave a line space between the header comment and the actual method signature.